lib/jellyfish_web/views/component_view.ex

```diff
@@ -0,0 +1,25 @@
+defmodule JellyfishWeb.ComponentView do
```

Comment:
After reading https://hexdocs.pm/phoenix/views.html#rendering-json, I don't see how we can benefit from using Views for returning JSON. In general, we don't want to render data, we just want to return JSON to the user that will execute its requests via curl, postman etc. 

Instead, I would favor implementing Jason.Econder for needed structs https://github.com/michalmuskala/jason#encoders

Using views for returning JSON seems to me like an overkill, results in a lot of boilerplate, and requires from us manual changes when Component struct changes (e.g. when adding fields)

We can ask someone else, someone who is more experienced in Phoenix